eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <!-- Keep this project at net6.0 until there is an SDK with net7.0 runtimes -->
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
The SDK may contain the correct runtimes but `@(KwownFrameworkReference)` items haven't been updated accordingly.
```suggestion
    <!-- Keep this project at net6.0 until there is an SDK with net7.0 runtimes -->
    <TargetFrameworks>net6.0</TargetFrameworks>
```